lib/translations/container.ex

```diff
@@ -0,0 +1,90 @@
+defmodule Scenic.Translations.Container do
+
+  @viewport :display_o_matic
+            |> Application.get_env(:viewport)
+            |> Map.get(:size)
+  # Takes a list of components/primitives, size of the container, options: starting x, y
+  def build_container(graph, [], starting_location), do: graph
+  def build_container(
+        %{
+          module: Scenic.Primitive.Rectangle,
+          data: {container_sizex, container_sizey} = container,
+          transforms: %{translate: {container_locationx, container_locationy}}
+        } = graph,
+        [%{styles: %{height: component_height, width: component_width}} = hd | rest],
```

Comment:
Unfortunately this is not the way a primitive will come across. The graph map will have a map of primitives.